### PR TITLE
Update idaes-pse requirement for IDAES 2.6

### PR DESCRIPTION
**Pull request by lbianchi-lbl**
## Summary/Motivation

Test idaes-pse 2.6.0rc0

## Reviewer's checklist / merge requirements:
- [x] The head branch (i.e. the "source" of the changes) is not the `main` branch on the PR author's fork
- [ ] Documentation
- [ ] Tests
- [ ] Diagnostic tests for models

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.

2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

**Comment by ksbeattie** [comment]
@mollydougher could you take a look at these failures (in the nanofiltration tests)?

**Comment by andrewlee94** [comment]
@mollydougher One thing I note in the test logs is that there are a lot of warnings about flow_vol being fixed to a value outside the bounds (value=0, bound=1e-8). That might be part of the issue.

**Comment by adam-a-a** [comment]
@mollydougher If you haven't yet, maybe you can try looking out logger output for the difailtration example before this change to see assess how problematic it was to solve in the first place. I am wondering if this model was slightly unstable in the first place or if there is really a problem introduced here via IDAES and/or WaterTAP.

@andrewlee94 suggests applying the DiagnosticToolbox as well.

**Comment by mollydougher** [comment]
Unfortunately the model is generally poorly scaled and sensitive to changes. I was able to go through and methodically update the scaling factors using the DiagnosticsToolbox and the ScalingToolbox, and the flowsheet converges with the IDAES 2.6 requirements (see [here](https://github.com/mollydougher/prommis/blob/fix-idaes2.6-diaf-failures/src/prommis/nanofiltration/diafiltration.py)). Assuming the only error was the failed convergence of IPOPT, I believe the checks should pass now. 

@lbianchi-lbl Should I push a commit directly to this PR, or make a separate PR of my own?

**Comment by lbianchi-lbl** [comment]
> Unfortunately the model is generally poorly scaled and sensitive to changes. I was able to go through and methodically update the scaling factors using the DiagnosticsToolbox and the ScalingToolbox, and the flowsheet converges with the IDAES 2.6 requirements (see [here](https://github.com/mollydougher/prommis/blob/fix-idaes2.6-diaf-failures/src/prommis/nanofiltration/diafiltration.py)). Assuming the only error was the failed convergence of IPOPT, I believe the checks should pass now.
> 
> @lbianchi-lbl Should I push a commit directly to this PR, or make a separate PR of my own?

@mollydougher thanks a lot for looking into this. I think it's fine to push the commit here (if you run into permissions issues, let me know) as the changes are directly related to IDAES 2.6 which is the purpose of this PR.

File: src/prommis/nanofiltration/diafiltration.py [hunk 1]
```diff
@@ -98,6 +98,16 @@ def main():
...
     scaling.propagate_solution(scaled_model, m)
 
+    # TODO: add Boolean variable to calculate pump OPEX
+    # Verify the feed pump operating pressure workaround is valid
+    # assume this additional cost is less than half a cent
+    if value(m.fs.feed_pump.costing.variable_operating_cost) >= 0.005:
+        raise ValueError(
+            "The variable   operating cost of the feed pump as calculated in the feed"
+            "pump costing block is not negligible. This operating cost is already"
+            "accounted for via the membrane pressure drop specific energy consumption."
```
Comment (agarciadiego) [inline src/prommis/nanofiltration/diafiltration.py:108]:
```suggestion
            "accounted for via the membrane's pressure drop specific energy consumption."
```

**Review by agarciadiego — APPROVED** [review]
Looks good to me, just a couple of typos before it gets merged

**Pull request merged**